virt/virt-4-8-release-notes.adoc

```diff
@@ -278,3 +278,8 @@ Error creating virtual machine
 *** As a workaround, create the virtual machine from the CLI or upgrade to {VirtProductName} 4.8.
 +
 (link:https://bugzilla.redhat.com/show_bug.cgi?id=1977037[*BZ#1977037*])
+
+//New known issue for 4.8.2
+* If an `nmstate-handler` pod restarts, or if you update a `NodeNetworkConfigurationPolicy` manifest that creates a Linux bridge interface, the desired network policy configuration fails to be applied. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2001901[*BZ#2001901*])
```

Comment:
This BZ belongs to 4.8.3 which might have the fix. Bugzilla for 4.8.2 is https://bugzilla.redhat.com/show_bug.cgi?id=2001796

Reply:
@geetikakay The BZ link is to track the engineering bug associated with this issue. We don't point to the documentation bug in the release notes.